pkgs/development/libraries/zookeeper_mt/default.nix

```
@@ -1,21 +1,64 @@
-{ stdenv, zookeeper, bash }:
+{ stdenv
+, fetchurl
+, autoreconfHook
+, autoconf
+, automake
+, gettext
+, jre
+, libtool
+, openssl
+, pkgconfig
+# We depend on ZooKeeper for the Jute compiler.
+, zookeeper
+}:
 
 stdenv.mkDerivation rec {
-  name = "zookeeper_mt-${stdenv.lib.getVersion zookeeper}";
+  pname = "zookeeper_mt";
+  version = stdenv.lib.getVersion zookeeper;
 
-  src = zookeeper.src;
+  src = fetchurl {
+    url = "mirror://apache/zookeeper/${zookeeper.pname}-${version}/apache-${zookeeper.pname}-${version}.tar.gz";
+    sha512 = "16994067d460a1b6af6a71f3458c64ee32629e876a1ff6646d57be62f1a5adab57462af84074ecaded4186dd3fde035ee24cd9d578b8e5044073eb05f4ab9c3e";
+  };
+
+  sourceRoot = "apache-${zookeeper.pname}-${version}/zookeeper-client/zookeeper-client-c";
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    autoreconfHook
+    gettext
+    jre
+    libtool
```

```suggestion
    autoreconfHook
    jre
```

The hook should already contain them.

---

Done.